tests/filesystems/test_config.py

```diff
@@ -73,6 +73,64 @@ def mapped_file(test_file: str, fs_unix: VirtualFilesystem) -> VirtualFilesystem
                 },
             },
         ),
+        (
```

Comment:
The test data needs to be adjusted when incorporating the suggested changes.

Reply:
Adjusted